src/app/components/IconSvg/IconSvg.module.css

```
@@ -0,0 +1,15 @@
+.svg {
+  width: 2em;
+  height: 2em;
+  fill: var(--color-inactive);
+}
+
+.svg:hover,
+.selected,
+.selected:hover {
+  fill: url(#occtyGradient);
```

```suggestion
  fill: url(#octyGradient);
```
It's from the word "octopus" with one `c`.